orm_lib/inc/drogon/orm/QueryBuilder.h

```diff
@@ -0,0 +1,45 @@
+/**
+ *
+ *  @file QueryBuilder.h
+ *  @author Ken Matsui
+ *
+ *  Copyright 2022, Ken Matsui.  All rights reserved.
+ *  https://github.com/drogonframework/drogon
+ *  Use of this source code is governed by a MIT license
+ *  that can be found in the License file.
+ *
+ *  Drogon
+ *
+ */
+
+#pragma once
+
+#include <drogon/orm/FilterBuilder.h>
+#include <string>
+
+namespace drogon
+{
+namespace orm
+{
+template <typename T>
+class QueryBuilder : public FilterBuilder<T, true>
+{
+  public:
+    inline QueryBuilder from(const std::string& table)
```

Comment:
I think the from() method is not needed, because the table name can be obtained by T::tableName.

Reply:
Thank you! I thought it can be kept for future scalability because I'd also like to improve the way models are organized. What about doing the following?

```cpp
inline FilterBuilder<T, false> select(const std::string& columns)
{
    // When users do not set the table name explicitly, then retrieve it from model T.
    return {this->from_.empty() ? T::tableName : this->from_, columns};
}
```